src/shared-components/selectorButton/index.tsx

```diff
@@ -76,10 +76,10 @@ export const SelectorButton = ({
         </SelectorIcon>
         <Selector
           type={type}
-          checked={checked}
+          selectorChecked={checked}
```

Comment:
One downside is that our `@testing-library/react` snapshots will remove any invalid HTML attributes we're applying to DOM nodes. In this case I noticed these two props being passed to a div that `react-test-renderer` serialized. We're only using these for styling, and we've been cleaning these up as we encounter them. It might be the case that we clean up fewer of these using `@testing-library/react`, but they're mostly a [nit] anyway so I do not think it should be weighed as a reason to keep `react-test-renderer`.